segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/SegmentStateStore.java

```diff
@@ -133,7 +139,7 @@ private ByteArraySegment serialize(SegmentState state) {
         }
     }
 
-    @SneakyThrows(IOException.class)
+    @SneakyThrows
```

Comment:
Why this change

Reply:
Because this method now throws `DataCorruptionException` in addition to `IOException`. AFAIK, there's no way of having `@SneakyThrows` declared multiple times.

Reply:
DataCorruptionException should be runtime by our definitions...